src/TRestRawSignalChannelActivityProcess.cxx

```diff
@@ -293,3 +297,19 @@ void TRestRawSignalChannelActivityProcess::EndProcess() {
 #endif
     }
 }
+
+///////////////////////////////////////////////
```

Comment:
We should avoid implementation of `InitFromConfigFile` for basic std (double,string,vector) type parameters. `InitFromConfigFile` should be reimplemented when a complex structure is required, and some logic must be imlemented.

The present `InitFromConfigFile` can be automatically read by `TRestEventProcess::InitFromConfigFile`, thus it is not needed.

There is an additional problem, for example, in the implementation of `daqChannels` that fills the metadata member `fDaqHistogramChannels`.

If this method is not reimplemented, then the naming convention will follow: `fDaqHistogramChannels` --> `daqHistogramChannels`, or `fDaqChannels` will be `daqChannels`.

Reply:
Ok, if `InitFromConfigFile `is not necessary, then all this PR is also not necessary. The problem is that the values in the RML were not being read, that's why I did this.
Now doing some other tests I found that they are read but not as expected, so I'm going to try and find the problem.

Reply:
I found a problem but I don't know how to solve it... Using the default values, it all works: 
![image](https://user-images.githubusercontent.com/77193103/180959037-b6965bc1-c3b7-4877-b2b5-db33167f21b4.png)

But if I introduce values manually in the RML, e.g.
```
<addProcess type="TRestRawSignalChannelActivityProcess" name="rawChActivity" value="ON" verboseLevel="info" >
    <parameter name="lowThreshold" value="20" />
    <parameter name="highThreshold" value="40" />
    <parameter name="daqChannels" value="400" /> 
    <parameter name="daqStartChannel" value="4000" /> 
    <parameter name="daqEndChannel" value="5000" />
    <parameter name="readoutChannels" value="250" />
    <parameter name="readoutStartChannel" value="1" />
    <parameter name="readoutEndChannel" value="300" />
 </addProcess>
```

then all the values are wrongly read and/or applied to the wrong variable, and some are taken from the default:
![image](https://user-images.githubusercontent.com/77193103/180960971-21a108e4-32b4-4f4f-8bd3-3efa78efd587.png)

Reply:
@jgalan do you  know why this might be happening?